monitor/main.go

```diff
@@ -177,6 +177,10 @@ func blockProcessor(ctx context.Context, startBlock *big.Int, headers <-chan *ty
 
 		transactionCtx := context.Background()
 
+		// Will be populated when creating contract processors
+		eventHandlers := make(map[registry.ContractID]EventHandler)
```

Comment:
this is nice, thanks for rectifying what I broke :)

Reply:
it could be interesting in future to add this functionality to kliento for installing custom handlers to specific logs, kinda like event hooks